Sources_v3/Workers/Background/ChannelWatchStateUpdater.swift

```diff
@@ -30,7 +30,7 @@ final class ChannelWatchStateUpdater<ExtraData: ExtraDataTypes>: Worker {
         webSocketConnectedObserver = WebSocketConnectedObserver(
             notificationCenter: webSocketClient.eventNotificationCenter,
             filter: { $0.connectionStatus == .connected },
-            callback: { [weak self] in self?.watchChannels() }
+            callback: { [unowned self] in self.watchChannels() }
```

Comment:
That's not required but I'd like to see the crash if the lifecycle of the internal observer doesn't follow the lifecycle of a worker